packages/documentation/pages/usage/components/button.vue

```diff
@@ -157,7 +169,8 @@
 
 <script lang="ts">
 import { KtButton } from '@3yourmind/kotti-ui'
-import { defineComponent } from '@vue/composition-api'
+import { KottiButton } from '@3yourmind/kotti-ui/dist/esm/kotti-button/types'
```

Comment:
Why importing from '@3yourmind/kotti-ui/dist/esm/kotti-button/types'?
shouldn't it be?
```
import { Kotti } from '@3yourmind/kotti-ui'
```
and then usage:
```
Kotti.Button.ToggleStatus.OFF
```